tests/test_cli.py

```diff
@@ -508,6 +508,78 @@ def test_eval_param(tmpdir):
     assert float(recall.group(1)) == 0.0
 
 
+def test_eval_resultsfile(tmpdir):
+    tmpdir.join('doc1.txt').write('doc1')
+    tmpdir.join('doc1.key').write('dummy')
+    tmpdir.join('doc2.txt').write('doc2')
+    tmpdir.join('doc2.key').write('none')
+    tmpdir.join('doc3.txt').write('doc3')
+    result = runner.invoke(
+        annif.cli.cli, [
+            'eval', '--results-file', 'test_file.txt', 'dummy-en',
```

Comment:
This test leaves behind (and clobbers if it exists) a file called `test_file.txt`. It would be better to use the `tmpdir` fixture also for this purpose, then the file would be automatically deleted afterwards:

* Create a [py.path.local](https://py.readthedocs.io/en/latest/path.html#py-path-local-local-file-system-path) object like this: `resultfile = tmpdir.join('results.tsv')`
* You can get the actual file path using `str(resultfile)` (for the `--results-file` parameter value)
* You can open it using `resultfile.open()`, or just call `.readlines()` on it